docs/user/user/pings/testing-custom-pings.md

```diff
@@ -1,300 +1,11 @@
-# Unit testing Glean custom pings for Android
+# Testing custom pings
```

Comment:
It could be interesting to add a paragraph or two on the importance of testing custom pings. That can be done as a follow up though.

Reply:
I think that's better material for https://mozilla.github.io/glean/book/user/pings/index.html

Reply:
Agree!

Reply:
Would you mind filing a bug for it? (Sorry if you already did it)

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1716800